packages/tools/examples/segmentSelect/index.ts

```diff
@@ -115,7 +115,9 @@ function setupTools(toolGroupId, isContour = false) {
   addManipulationBindings(toolGroup);
 
   // Segmentation Tools
-  toolGroup.addTool(SegmentSelectTool.toolName);
+  toolGroup.addTool(SegmentSelectTool.toolName, {
+    mode: SegmentSelectTool.SelectMode.Inside,
```

Comment:
I envision this as a completely separate tool, maybe called `segmentLabel` or something similar; it's really unrelated to `SegmentSelectTool`.

Reply:
Done.